homework1/liang/.gitignore

```diff
@@ -0,0 +1,24 @@
+.idea/
```

Comment:
Usually personalized files should not ignore by project based .gitignore but personal .gitignore in your home directory.
for example other developer might not using intellij. .idea may not make sense to them